Certified Sail Extension (CSE) Examples/cse/mapField/v1/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel='stylesheet' href='./map.css'></link>
+</head>
+
+<body>
+  <div id='map-container'></div>
+  <script src='APPIAN_JS_SDK_URI'></script>
+  <script>
+    var mapState = {};
+    var newKey, newLocation, currentKey;
+    var scriptId = 'scriptTag';
```

Comment:
We should make this id something more descriptive of that this is:
`var mapsApiScriptId = 'google-maps-api-script';`